lib/hammer_cli_foreman/host.rb

```diff
@@ -66,6 +67,8 @@ def extend_data(host)
         field nil, _("Puppet Master Proxy"), Fields::SingleReference, :key => :puppet_proxy
         field :certname, _("Cert name")
         field :managed, _("Managed"), Fields::Boolean
+        field :build_status_label, _("Build Status")
+        field :global_status_label, _("Global Status")
```

Comment:
Would you mind moving the global status above the build one? I'd say it makes more sense to see it on the first place.